index/image__vector_store.json

```diff
@@ -0,0 +1 @@
+{"embedding_dict": {}, "text_id_to_ref_doc_id": {}, "metadata_dict": {}}
```

Comment:
the file has a double underscore in the name here ```image__``

Reply:
image__vector_store.json was generated when serialising index. We should keep the name since the loader expects this name.